pyogrio/_ogr.pxd

```diff
@@ -33,6 +34,17 @@ cdef extern from "cpl_string.h":
     char**      CSLAddString(char **list, const char *string)
 
 
+cdef extern from "cpl_vsi.h" nogil:
+
+    ctypedef int vsi_l_offset
```

Comment:
I don't know that we specifically need `ctypedef int vsi_l_offset`; can just use `int` in place of it in `VSIFileFromMemBuffer`.

(unlike other typedefs that we include here because they provided added-value to clarify a parameter, like `VSILFILE`)

Reply:
Yes, will clean this up (this was copied from fiona)